src/Question.js

```diff
@@ -1,49 +0,0 @@
-import React from "react";
-import Header from "./Header";
-
-const Question = ({ quizQuestions, quizOptions, images, setQuesAnswered, callBack, id, choice }) => {
-
-    return (
-        <>
-            <Header />
-            <div className="container">
-                <div className="heading2Div">
-                    <h2> {quizQuestions} </h2>
-                </div>
-                <div className="options" >
-                    {
-                        quizOptions.map((value) =>
-                        (<>
-                            <div className="selectOptions">
-                                <input key={value}
-                                    type="radio"
-                                    id={id}
-                                    name="radioBtn"
-                                    value={value}
-                                    onChange={
-                                        () => { callBack(value); console.log(value); }}
```

Comment:
it is difficult to understand what this code is doing. Can we change this value to `updateAnswer` instead, this way we know what the action is meant to be doing

Reply:
hmm - I don't  think I understand the intent of `updateAnswer` enough. Can you please explain what this function is used for?

Reply:
So, in the App component I have defined three variables namely - mood, location and socialize. I am using useState to change the value of the variables. So when i am passing data to the question component I am passing the callback functions of the above mentioned variables. Once the user selects an option in the input field, onChange eventlistener sets the value of the options(mood,location,socialize).